CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 
 ---
+## [X.X.X]
```

Comment:
you still have some unit tests errors. 

didn't manage to install swiftymocky  ?